docs/typescript.md

```diff
@@ -138,16 +138,16 @@ Out of the box, transforming your files to JavaScript works via the same [Babel
 
 ## What does React Native + TypeScript look like
 
-You can provide an interface for a React Component's [Props](props) and [State](state) via `React.Component<Props, State>` which will provide type-checking and editor auto-completing when working with that component in JSX.
+You can provide an interface for a React Functional Component's [Props](props) via `React.FC<Props>` which will provide type-checking and editor auto-completing when working with that component in JSX.
```

Comment:
```suggestion
You can provide an interface for a React Function Component's [Props](props) via `React.FC<Props>` which will provide type-checking and editor auto-completing when working with that component in JSX.
```

It's "function" component, not "functional" - https://reactjs.org/docs/components-and-props.html.